multimodal-search-pdf/requirements.txt

```diff
@@ -1,5 +1,7 @@
-jina[hub, http]==1.2.0
+jina[hub, http]==2.0.0rc3.dev24
```

Comment:
You might potentially upgrade to 2.0.0rc5.dev70 - not super important probably but some things have changed

Reply:
Since the example will be moved to legacy, it will become quickly outdated again. I would opt for leaving it as it is.